x-pack/platform/plugins/shared/stack_connectors/common/openai/types.ts

```diff
@@ -18,12 +18,20 @@ import type {
   InvokeAIActionResponseSchema,
 } from './schema';
 
-export type Config = TypeOf<typeof ConfigSchema>;
+export type Config = TypeOf<typeof ConfigSchema> & {
+  certificateFile?: string | string[];
+  certificateData?: string;
+  privateKeyFile?: string | string[];
+  privateKeyData?: string;
+  verificationMode?: 'full' | 'certificate' | 'none';
+  caFile?: string | string[];
+  caData?: string;
+};
```

Comment:
```suggestion
export type Config = TypeOf<typeof ConfigSchema>;
```

Revert this, there is no need to do this since these properties are already part of the `ConfigSchema`